### ISSUE TITLE
Optinally remove playroom preview animation

### ISSUE THREAD

**Issue by maraisr** [seek-oss/playroom#159]
I'm currently developing a component that I'm needing to emulate inside of the iOS simulator. To combat some iOS specific issues. Anyway - so Im using the preview mode, but my HMR is causing it to refresh from the start, and the playroom animation kicks on for about 1-3seconds, which for style changes is quite annoying.

I aim to introduce either a config, or simply url param to disable this fulls screen animation from running, and to just render my component at whatever internal it has to. Ie dont wait for it to finish before your animation finishes.

**Comment by possibilities** [comment]
Maybe like the frame component it could be configurable with the current animation as default? Then people could set it to `() => null` to disable it? Happy to contribute this if helpful.

**Comment by possibilities** [comment]
Actually it's hard to tell if it's added for a functional reason from looking at the code? Maybe be a better solution depending on that.

**Comment by maraisr** [comment]
yeah I mean at this stage just have to wait for a response from @mattcompiles @michaeltaranto @markdalgleish for an answer, or guidance.

**Comment by michaeltaranto** [comment]
We actually started off setting this up as a `PreviewComponent` that you could override with a passthrough if you liked. The reason we changed the approach was due to security concerns—with prototypes essentially being used for phishing attacks.

Preview mode is not really intended for the fast iteration on the prototype itself, however maybe there is value in only showing the splash screen in the built artefact rather the dev server?

**Comment by possibilities** [comment]
One idea I had would be to remove the preview altogether but pass a prop into the frame component e.g. `isPreviewing` and if desired folks could inject whatever loading screen, etc they desire in their frame?

**Comment by maraisr** [comment]
> however maybe there is value in only showing the splash screen in the built artefact rather the dev server?

@michaeltaranto  I think that approach works really nicely. This way development time is quick, and a developer knows that it wont be 100% perfect.

Is this something I can help bring to playroom? @markdalgleish has expressed the plate, and how its always full for you guys.

**Comment by askoufis** [comment]
The preview animation was replaced with a dismissable header in [1.0.0](https://github.com/seek-oss/playroom/releases/tag/v1.0.0).
